src/repart/repart.c

```diff
@@ -6098,7 +6098,8 @@ static int context_mkfs(Context *context) {
                         return r;
 
                 r = make_filesystem(partition_target_path(t), p->format, strempty(p->new_label), root,
-                                    p->fs_uuid, arg_discard, /* quiet = */ false,
+                                    p->fs_uuid, arg_discard,
+                                    /* quiet = */ streq(p->format, "erofs") && !DEBUG_LOGGING,
                                     context->fs_sector_size, p->compression, p->compression_level,
```

Comment:
also, please add: `// FIXME: should be dropped once mkfs.erofs has been fixed to become less noisy` or so